Gruntfile.js

```diff
@@ -15,24 +15,22 @@ module.exports = function (grunt) {
 	grunt.loadNpmTasks('grunt-contrib-watch');
 	grunt.loadNpmTasks('grunt-retire');
 	grunt.loadNpmTasks('grunt-mocha');
+	grunt.loadNpmTasks('grunt-prettier');
 	grunt.loadTasks('build/tasks');
 	grunt.loadNpmTasks('grunt-parallel');
 
 	var langs;
 	if (grunt.option('lang')) {
-		langs = (grunt.option('lang') || '')
-			.split(/[,;]/g).map(function (lang) {
-				lang = lang.trim();
-				return (lang !== 'en' ? '.' + lang : '');
-			});
-
+		langs = (grunt.option('lang') || '').split(/[,;]/g).map(function(lang) {
+			lang = lang.trim();
+			return lang !== 'en' ? '.' + lang : '';
+		});
 	} else if (grunt.option('all-lang')) {
 		var localeFiles = require('fs').readdirSync('./locales');
-		langs = localeFiles.map(function (file) {
+		langs = localeFiles.map(function(file) {
```

Comment:
This change breaks syntax highlighting in Sublime

Reply:
You may want to use JavaScript (Babel). The standard one doesn't deal with ES6 very well.